common/src/main/java/com/wynntils/wc/custom/item/UnidentifiedItemStack.java

```diff
@@ -30,7 +29,7 @@
 
 public class UnidentifiedItemStack extends WynnItemStack {
     private final List<Component> tooltip;
-    private ItemType itemType;
+    private Optional<ItemType> itemType;
```

Comment:
I think optionals are not meant to be fields. Make this an ItemType, but keep `getItemType` an Optional (and return empty if null)